### PR TITLE
Adds atomic footer

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Updated footer to be atomic.

## Testing

- run `gulp scripts:ondemand && styles:ondemand` and check `http://localhost:8000/test-fixture/?atomic=footer`
- run `gulp build` and site footer on the site should look/function the same (return to top at mobile should work).

## Review

- @Scotchester 
- @rosskarchner 
- @sebworks 
- @KimberlyMunoz 
- @jimmynotjim 

## Screenshots

screen shot 2016-04-06 at 1 16 22 pm

### CODE REVIEW

File: cfgov/unprocessed/css/organisms/footer.less [hunk 1]
```diff
@@ -150,62 +150,63 @@
...
             .grid_column(8);
             border-right: 1px solid @gray-40;
             border-left: 0;
 
-            .footer_col {
+            .o-footer_col {
                 .grid_column(6);
                 border-left: 0;
                 border-right: 0;
+                padding-right: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
```
Comment (anselmbradford) [inline cfgov/unprocessed/css/organisms/footer.less]: This is to take care of this issue:

screen shot 2016-04-06 at 1 01 29 pm

File: cfgov/jinja2/v1/_includes/organisms/footer.html [hunk 2]
```diff
@@ -55,12 +55,12 @@
                         block__padded-top">
                 {{ social_media.render( { 'is_share_view': false } ) }}
             </div>
-        </div><!-- END .footer-pre -->
+        </div><!-- END .o-footer_pre -->
```
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/footer.html]: Could we change this to a jinja comment while we're in this file?

Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/organisms/footer.html]: The deed is done https://github.com/cfpb/cfgov-refresh/pull/1837/commits/f40911286897766217a845317af9c772f357ae13

**Comment by jimmynotjim** [comment]
:+1: Nice work on these on-demand bits, this is going to make this work a heck of a lot easier.

**Comment by sebworks** [comment]
:+1:

**Pull request merged**